apps/src/componentLibrary/slider/Slider.tsx

```diff
@@ -52,29 +101,223 @@ const Slider: React.FunctionComponent<SliderProps> = ({
   name,
   value,
   disabled = false,
-  size = 'm',
-  ...HTMLAttributes
+  color = 'black',
+  isCentered = false,
+  isPercentMode = false,
+  step = 1,
+  steps,
+  isRtl = document.documentElement.dir === 'rtl',
+  minValue = 0,
+  maxValue = 100,
+  leftButtonProps,
+  rightButtonProps,
+  className,
+  ...HTMLInputAttributes
 }) => {
-  const bodyTextSize = componentSizeToBodyTextSizeMap[size];
+  const labelId = `${name}-label`;
+  const fillColor = sliderTrackColorsMap[color][disabled ? 'disabled' : 'fill'];
+  const emptyColor =
+    sliderTrackColorsMap[color][disabled ? 'disabled' : 'empty'];
+  const gradientDirection = isRtl ? 'left' : 'right';
+
+  const [backgroundStyle, setBackgroundStyle] = useState('');
+
+  // Override min and max values for percent mode
```

Comment:
If `isPercentMode` is true, should we enforce that min and max slider value are undefined? I guess it's not required but it might be good to at least include a comment that if `isPercentMode` is true, then `minSliderValue` and `maxSliderValue` will be ignored?

Reply:
I don't think we should set a strict limit here since I believe that theoretically we might want any range, including negative value and possibly more than 100 in both directions.
The only difference with percent mode is that it is showing the `%` sign next to the current input value when enabled